testutils/common/data.common.testutil.js

```diff
@@ -51,14 +51,30 @@ function generateClientUser(overrides = {}) {
  */
 function generateUsers(
   count,
-  { public: pub, locale } = {},
+  {
+    additionalProvidersData,
+    extSitesBW,
+    extSitesCS,
+    extSitesWS,
+    locale,
+    newsletter,
+    public: pub,
+  } = {},
```

Comment:
Was a bit surprised this doesn't just take anything, but passes a specific list of keys allowed for modification. 🤷‍♂️